packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Return rejected promises instead of throwing
```

Comment:
Changelog message should be more specific - this isn't a change that's happened everywhere, just in one part of resin-universal-ga.